### PR TITLE
kurtosis ci to trigger on docker build/push workflow

### PR DESCRIPTION
**Pull request by sudeepdino008**
- this PR experiments with a 2nd experimental version of docker build workflow...it eventually deletes that
- the  (original) docker build workflow is changed to invoke kurtosis-assertoor workflow, once it succeeds.
- here's a [sample run](https://github.com/erigontech/erigon/actions/runs/11105470450/job/30851703289) - this had the experimental workflow call the kurtosis ci...

### CODE REVIEW

File: cmd/rpcdaemon/README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Example request
 `http POST http://localhost:8545/health --raw '{"min_peer_count": 3, "known_block": "0x1F"}'`
 Example response
 
-```
```
Comment (sudeepdino008) [inline cmd/rpcdaemon/README.md:106]: unrelated change which was needed to invoke make docker image workflow (which didn't get invoked only on commit changes in .github/ dir)

**Review by somnathb1 — APPROVED** [review]
